config/checkstyle_checks.xml

```diff
@@ -402,7 +402,7 @@
     <module name="InnerAssignment"/>
     <module name="MagicNumber"/>
     <module name="MatchXpath">
-      <property name="query" value="//CLASS_DEF[@text!='Checker' and @text!='Main']
+      <property name="query" value="//COMPILATION_UNIT/CLASS_DEF[@text!='Checker' and @text!='Main']
```

Comment:
Do we really need this update ?
We do not care where class is located. - `//`

If we have `/` let's change it to `//` . And let put this as migration note to users as it breaking compatibility :) for Xpath.

Reply:
Done.

Reply:
@nmancus1 , let use double slash, to keep xpath compact

Reply:
> let use double slash, to keep xpath compact

In migration note?

Reply:
In this PR.
I do not see a reason to overcomplicate Xpath.

Adding root node supposed to be invisible to users, but looks like it is not.

It ok to keep root node for cases where we use generated value of Xpath, but all others should be compact.

Reply:
This line ( that is target of this review thread) should not be changed at all .

Reply:
I have also changed `/COMPILATION_UNIT/CLASS_DEF` to `//CLASS_DEF` wherever possible. Done.